ncm-opennebula/src/main/resources/group.tt

```diff
@@ -1,10 +1,11 @@
-[% FOR pair IN group.pairs -%]
-[%-    SWITCH pair.key -%]
-[%         CASE 'description' -%]
-[%             pair.key FILTER upper %] = "[% pair.value %]"
-[%         CASE 'group' -%]
-[%             IF pair.value != 'oneadmin' -%]
+[% FOR pool IN group.pairs -%]
+[%     FOR pair IN pool.value.pairs %]
+[%-        SWITCH pair.key -%]
+[%             CASE 'description' -%]
```

Comment:
Same reason than the user case, maybe we can replace the `SWITCH` by `IF` statements.. but I'm not sure if we will need more conditions in the future